### DISCUSSION TITLE
Sandboxing separate instances of browser, to facilitate VPN split tunneling

### DISCUSSION THREAD

**Discussion by e-t-l** [sandboxie-plus/Sandboxie#4056] [Q&A Feedback] [answered]
I currently have a system-wide VPN that allows split-tunneling, so I can specify programs by path to whitelist (i.e. circumvent VPN). I currently have Firefox installed on the host machine and forced to run in a sandbox. I am hoping to create a new sandbox containing a Firefox executable (which I can then whitelist in my VPN). I am hoping this whitelisted executable can use all the installation files of the original host-machine Firefox installation, so I don't have to fully install a Firefox instance within the sandbox.

I tried creating a sandbox, duplicating the directory structure "drive\C\Program Files\Mozilla Firefox\", and copying "firefox.exe" into that sandbox folder. In the sandbox settings, I added the Firefox template so it can use all the host machine Firefox installation files, but I added `ClosedFilePath=C:\Program Files\Mozilla Firefox\firefox.exe`. I hoped this would make it use the copied Firefox executable but otherwise utilize all the other Firefox installation files located in the real "C:\Program Files\Mozilla Firefox" folder.

It didn't work. I got a Windows error "The application has failed to start because its side-by-side configuration is incorrect." Does anyone know if my goal is achievable? I'm thinking the other thing I could try is to make hard-symlinks of the entire contents of "C:\Program Files\Mozilla Firefox" in "C:\Sandbox\Username\noVPN\drive\C\Program Files\Mozilla Firefox\"...

**Comment by offhub** [comment, marked as answer]
Try it after copying the `mozglue.dll` file.

Or, create a file named `firefox_novpn.exe` in the host location to be used in the split tunnel.

```ini
SpecialImage=firefox,firefox_novpn.exe
```

**Reply by e-t-l** [reply]
> Or, create a file named firefox_novpn.exe in the host location to be used in the split tunnel.

😖🤦‍♂️ that's so much more straightforward, thank you
